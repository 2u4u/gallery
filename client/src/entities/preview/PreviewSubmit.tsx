type Props = {
    handleUpload: () => void;
}

function PreviewSubmit({ handleUpload }: Props) {
    return (
        <div className='preview__block preview__block--footer'>
            <button type='button' onClick={handleUpload} className={'preview__button'} >
                {'Upload'}
            </button>
        </div>
    );
}

export default PreviewSubmit;
