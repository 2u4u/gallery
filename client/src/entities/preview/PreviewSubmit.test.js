import renderer from 'react-test-renderer';
import PreviewSubmit from './PreviewSubmit';

describe('Preview submit component ', () => {
    const handleUpload = () => false;
    it('matches snapshot', () => {
        const component = renderer.create(
            <PreviewSubmit handleUpload={handleUpload} />,
        );
        let tree = component.toJSON();
        expect(tree).toMatchSnapshot();
    });
});
