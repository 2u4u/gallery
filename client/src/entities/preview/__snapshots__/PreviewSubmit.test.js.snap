// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Preview submit component  matches snapshot 1`] = `
<div
  className="preview__block preview__block--footer"
>
  <button
    className="preview__button"
    onClick={[Function]}
    type="button"
  >
    Upload
  </button>
</div>
`;
